libsolidity/parsing/Token.cpp

```diff
@@ -132,6 +132,7 @@ tuple<Token::Value, unsigned int, unsigned int> Token::fromIdentifierOrKeyword(s
 		Token::Value keyword = keywordByName(baseType);
 		if (keyword == Token::Bytes)
 		{
+			solAssert(m != -1, "Invalid type M in fixed command. Should not reach here.");
```

Comment:
I think `bytes888888888888888888888888888888888888` will trigger this assert, which should not happen (it is a valid identifier).